vm/src/object/ext.rs

```diff
@@ -214,7 +214,10 @@ impl<T: PyPayload> ToPyObject for PyRefExact<T> {
     }
 }
 
-pub struct PyAtomicRef<T: PyObjectPayload>(PyAtomic<*mut Py<T>>);
+pub struct PyAtomicRef<T> {
+    inner: PyAtomic<*mut i8>,
```

Comment:
```suggestion
    inner: PyAtomic<*mut u8>,
```
it doesn't look important, but i've usually seen other code use u8 for this case. (to express `void*`)